zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -289,52 +233,78 @@ impl<F: Field> ExecutionConfig<F> {
 
             iter::once(sum_to_one)
                 .chain(bool_checks)
-                .chain(execution_state_transition)
-                .map(move |(name, poly)| (name, q_step.clone() * poly))
-                // TODO: Enable these after test of CALLDATACOPY is complete.
-                // .chain(first_step_check)
-                // .chain(last_step_check)
+                .map(move |(name, poly)| (name, q_usable.clone() * q_step.clone() * poly))
+            // TODO: Enable these after test of CALLDATACOPY is complete.
+            // .chain(first_step_check)
+            // .chain(last_step_check)
```

Comment:
There are `COPYTOMEMORY` and `COPYCODETOMEMORY` still being tested with incomplete trace. In #419 I removed them because I believe they are covered by `CALLDATACOPY` and `CODECOPY`, and enable these checks.

I think it's okay to just keep them as TODO in this PR.

Reply:
Ohhh I see.
Well, then let's file a `tech-debt` issue for this so that we don't forget no? 

WDYT?

Reply:
Sounds good to me! Just opened issue #464.